README.md

```diff
@@ -1 +1,9 @@
-# HACK-OHIO-2019
\ No newline at end of file
+# HACK-OHIO-2019
```

Comment:
I think this would be nicer if the "-" were " " and we put a slash between the "I" and "O".

The only reason it wasn't created that way was because it defaults to the repo name and repo names can't include "/" or " "